src/components/AdminHome/AdminHome.js

```diff
@@ -30,15 +32,15 @@ function AdminHome(props) {
             <h2>Welcome, 'username'</h2>
             <div class='display'>
             <div class="adminNav">
-                    <h5 class="navText">Existing WorkFlows</h5> <h5 onClick={existingProjects} class="navText">Existing Projects</h5> <h5 onClick={clientAlerts} class="navText">Client Alerts</h5> <h5 onClick={clientList} class="navText">Client List</h5> 
+                    <h5 onClick={existingWorkflows} class="navText">Existing WorkFlows</h5> <h5 onClick={existingProjects} class="navText">Existing Projects</h5> <h5 onClick={clientAlerts} class="navText">Client Alerts</h5> <h5 onClick={clientList} class="navText">Client List</h5> 
             </div>
             <ul>
-                <li onClick={newWorkflow}>New WorkFlow</li>
+                <li onClick={createWorkflow}>Create WorkFlow</li>
                 <li>New Project</li>
 
                 <li>Existing Projects</li>
             </ul>
```

Comment:
I think this ul and list items were initally for mocking up what would be in the nav bar. they can probably be deleted now